### DISCUSSION TITLE
What does it mean when an input/output is grayed out on the dashboard?

### DISCUSSION THREAD

**Discussion by fong1999** [concourse/concourse#8762] [Help & Support] [answered]
Hi! 

I noticed that on concourse dashboard, some of the inputs/outputs are not highlighted/grayed out. Any ideas what that means? I [linked an image here](https://miro.medium.com/v2/resize:fit:1400/format:webp/1*NhjjisyqNuCJL3vnzhBqCw.png). You see, the "resource-repo" in this case is grayed out. Why is that? 

Please let me know if you need more information. 

Thanks much!

**Comment by taylorsilva** [comment, marked as answer]
It's greyed out to signify there's a `passed` constraint on this resource for the next job it connects to in the pipeline. You can see this in Concourse's public pipeline as well where there are more resources being passed between jobs: https://ci.concourse-ci.org/teams/main/pipelines/concourse?group=k8s

![image](https://github.com/concourse/concourse/assets/4583837/7d26788b-6203-4529-a0da-7f06e72c7068)

**Reply by fong1999** [reply]
Thank you for the info!
